backend/graminsta/post/urls.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: UTF-8 -*-
+"""
+Urls for post app.
+"""
+
+from django.urls import path
+from .views import FollowView
+
+app_name = 'post'
```

Comment:
What is this?

Reply:
I learned it in Django Tutorial and it is said to be URL namespaces: https://docs.djangoproject.com/en/3.1/intro/tutorial03/#namespacing-url-names
See also:
https://stackoverflow.com/questions/52751549/django-2-0-7-how-url-app-name-works

Reply:
Also when I run the server I got this error since `core/url.py` did not provide `app_name` and `graminsta\urls.py` specified a namespace in include(), so I suggest that @shiyi001 add it too.
```
django.core.exceptions.ImproperlyConfigured: Specifying a namespace in include() without providing an app_name is not suppor
ted. Set the app_name attribute in the included module, or pass a 2-tuple containing the list of patterns and app_name inste
ad.

```

Reply:
OK, got it. I think that's something new for me. Back to the time when I wrote Django apps, there was no `app_name`.